README.md

```diff
@@ -99,7 +99,7 @@ graph TD;
     frontendApp-->|HTTP requests| backendAPI;
     backendAPI-->|Queries| database;
     backendAPI-->|Sends emails| emailService(CHES);
-    backendAPI-->|Sends marketing data| marketingService(Mautic);
+    backendAPI-->|Sends marketing data| marketingService(MailChimp);
```

Comment:
does our backend sends requests to MailChimp?

Reply:
our backend sends request to mailchimp, to update the list of users, shouldn't it b e reflected in redme?

Reply:
no, it is not backend API, removed from diagram